# Report

In [1]:
from dataset import *

## Data Preprocessing

In [11]:
!python3 dataset.py

loading train...
loading val...
loading test...
Train:	 300
Val:	 150
Test:	 100
Mean: [124.495 118.847  95.293]
Std:  [62.754 59.597 62.425]


### (a)
i). <br />
Mean: [124.495 118.847  95.293]<br />
Std:  [62.754 59.597 62.425]<br /><br />
ii). <br />
Because we are training a model that is representative of the training dataset. This means that the model should be fit based on properties of the training data. We are simply applying the model to validation and testing data; so if we use their properties, it may cause overfitting.

### (b)
![](Data_Preprocessing.png)

## Convolutional Neural Network

### (a)
There are 32*2 = 64 parameters to be learned; they are the weights of the fully connected layer.<br />

